<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="layerHirachyWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>721</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="createHierachy_Button">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>410</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Mannequin Hierachy</string>
    </property>
   </widget>
   <widget class="QLabel" name="topicLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>M11 Mannequin Hierachy Creator</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rigName_Selector">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rig Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="revert_Button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Revert Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
